package.json

```diff
@@ -1,15 +1,15 @@
 {
   "private": true,
-  "version": "0.1.0-alpha.1",
+  "version": "0.2.0",
```

Comment:
should start versioning at the top level to reflect changes to the repository structure. How do we know when it's at 1.0.0? I don't know